### ISSUE TITLE
Better documentation in ldc.conf

### ISSUE THREAD

**Issue by MrcSnm** [ldc-developers/ldc#4371]
Where can we get a better documentation in that file? I can't see anywhere what are the actual valid values in it and how they should be formatted, my usage from them was specifically from existing content, but it would be nice to understand what are the actual valid values with some descriptions

**Comment by JohanEngelen** [comment]
Can you be more specific?
What do you mean with "valid values"? (do you mean valid 'variables' like `switches`, `post-switches`, etc. ?)
What part of the formatting do you not understand from the existing default content?

**Comment by MrcSnm** [comment]
Yes, those variables like you. What I want is a formal documentation or specification on those variables. For example, I didn't knew there was a `post-switches`, and every time someone wants to cross compile to a different platform, I feel like there's much friction by not having those fairly documented

**Comment by kinke** [comment]
> For example, I didn't knew there was a `post-switches`

It's used by all `ldc2.conf` we generate here (based on the `ldc2*.conf.in` files in the repo). Everything I found useful is documented in the bundled files directly, including all 4 currently supported settings - there aren't any more.

**Comment by MrcSnm** [comment]
Well if those are the only settings in conf, guess it is enough then.
